Tests/SentryTests/SentryCrash/SentryDebugMetaBuilderTests.swift

```diff
@@ -0,0 +1,152 @@
+@testable import Sentry
+import XCTest
+
+/**
+ * Some of the test parameters are copied from debugging
+ * SentryCrashReportConverter.convertDebugMeta.
+ */
+class SentryDebugMetaBuilderTests: XCTestCase {
+    
+    private var imageProvider: TestSentryCrashBinaryImageProvider!
+    private var sut: SentryDebugMetaBuilder!
+    
+    override func setUp() {
+        super.setUp()
+        
+        imageProvider = TestSentryCrashBinaryImageProvider()
+        sut = SentryDebugMetaBuilder(binaryImageProvider: imageProvider)
+    }
+    
+    func testThreeImages() {
+        let imageName = "dyld_sim"
+        let imageNameAsCharArray = stringToUIntCharArray(value: imageName)
+        let image = createSentryCrashBinaryImage(
+            address: 4_386_213_888,
+            vmAddress: 140_734_563_811_328,
+            size: 352_256,
+            name: imageNameAsCharArray
+        )
+        
+        let actual = whenBuildDebugMetaWith(images: [image, image, image])
+        
+        XCTAssertEqual(3, actual.count)
+        XCTAssertEqual(actual[0].name, actual[1].name)
+        XCTAssertEqual(actual[1].name, actual[2].name)
```

Comment:
M: I don't see the necessity of this test.
I'd rather do a parameterized test with a given list, so the block below is run 3 times, asserting each item of the list

Reply:
Good point thanks 👍